app/controllers/medications_controller.rb

```diff
@@ -18,10 +18,21 @@ def create
     redirect_to '/dashboard'
   end
 
+  def edit
+    @medications = current_user.medications.all
+  end
+
+  def destroy
+    current_user.medications.destroy(med_params[:id])
+    Medication.destroy(med_params[:id])
```

Comment:
We need to remove this line in another PR. 
`Medication.destroy(med_params[:id])`